README.md

```diff
@@ -1,2 +1,27 @@
 # aks-auto-shutdown
 Config for Auto-Shutdown/Start of the AKS clusters
+
+In the near future, all environments excluding Production will be automatically shutdown. This action is to reduce the unnecessary infrastructure costs while the environments are not in use.
+
+## Default cluster shutdown hours
+
+20:00 to 06:30 everyday of the week.
+
+## Skip shutdown functionality
+
+In the event that an environment or environments are needed outside of the default hours, you can raise an "issue" to automatically exclude it from the shutdown schedule.
+- [Complete this issue form](https://github.com/hmcts/aks-auto-shutdown/issues/new?assignees=&labels=&projects=&template=shutdown_form.yaml).
+- Multile enviornments within the same "Business area" can be selected at the same time.
```

Comment:
```suggestion
- Multiple environments within the same "Business area" can be selected at the same time.
```

Reply:
This has the logic to loop over the json and skip shutdowns - although we aren't ready for that yet. So just parking with the name "future".